pandora-client-web/src/components/profileScreens/profileDescription.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ReactElement } from 'react';
+import { SpaceInviteEmbed } from '../../ui/screens/spaceJoin/spaceJoin';
+import { UntrustedLink } from '../common/link/externalLink';
+import React from 'react';
+
+export function ProfileDescriptionLine({ line, index }: { line: string; index: number; }): ReactElement {
+	if (line.match(/^https?:\/\//)) {
+		const url = new URL(line);
+		switch (url.hostname) {
+			case 'project-pandora.com':
+			case 'www.project-pandora.com':
+				if (url.pathname.startsWith('/space/join/')) {
+					const invite = url.searchParams.get('invite') ?? undefined;
+					const spaceId = url.pathname.split('/').pop();
+					if (!spaceId)
+						break;
+
+					return <SpaceInviteEmbed key={ index } spaceId={ spaceId } invite={ invite } />;
+				}
+				break;
+		}
+		return (
+			<UntrustedLink key={ index } href={ line }>
+				{ line }
+			</UntrustedLink>
+		);
+	}
```

Comment:
this is the same as in `RenderChatPart` we should only have a single function/component for this

Reply:
Yeah, I'd meant to ask about that, but I ended up forgetting. Somewhat unsure on where the best place to have that function would be

Reply:
maybe next in spacejoin where the `SpaceInviteEmbed` is, also not quite sure

Reply:
Changed to move the URL handling logic into a common function, part of it is still replicated, but was as close as i figured it should be given the slight differences between `RenderChatPart` and the added `ProfileDescriptionLine`.